src/process/_process-step.html

```diff
@@ -180,6 +180,8 @@ <h1 class="content-l_col-1-4 content-l_col h6 u-mb10">
                 </div>
                 <p class="short-desc block block__bg block__sub block__bg-small">
                   Visit our <a href="/owning-a-home/process/sources/">sources page</a> to learn more about the facts and numbers we reference.
+                 <br>
+                 <i>Note: The process and forms described on this page reflect <a href="/know-before-you-owe/" target="_blank">new mortgage regulations</a> that take effect for most mortgages on October 3, 2015.</i>
                 </p>
```

Comment:
If you want more spacing between the sources link & note, you could put them in separate paragraphs & add a parent element to apply the background:

```html
<aside class="short-desc block block__bg block__sub block__bg-small">
  <p>
    Visit our <a href="/owning-a-home/process/sources/">sources page</a> to learn more about the facts 
and numbers we reference.
  </p>
  <p class="u-mb0">
    <i>Note: The process and forms described on this page reflect <a href="/know-before-you-owe/" 
target="_blank">new mortgage regulations</a> that take effect for most mortgages on October 3, 2015.</i>
  </p>
</aside>
```